### PR TITLE
Add changelog for 0.72.0

### PR DESCRIPTION
**Pull request by rgrunber**
Signed-off-by: Roland Grunberg <rgrunber@redhat.com>

@fbricon , @testforstephen

### CODE REVIEW

**Review by testforstephen — APPROVED** [review]

**Comment by testforstephen** [comment]
Add https://github.com/redhat-developer/vscode-java/pull/1721 to changelog.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
 # Change Log
 
+## 0.72.0 (December 2nd, 2020)
+ * enhancement - Support inferSeletion when extract to variable. See [#1717](https://github.com/redhat-developer/vscode-java/pull/1717).
```
Comment (testforstephen) [inline CHANGELOG.md:4]: typos. inferSeletion

Support infer the selection range when extract to variable.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,15 @@
...
+ * bug fix - jdt.ls distro is 10MB heavier because of com.ibm.icu_64.2.0.v20190507-1337.jar. See [JLS#1351](https://github.com/eclipse/eclipse.jdt.ls/issues/1351).
+ * other - Update TP to use Eclipse 2020-12 M3. See [JLS#1616](https://github.com/eclipse/eclipse.jdt.ls/pull/1616).
```
Comment (testforstephen) [inline CHANGELOG.md:6]: Update target platform to use ...

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,15 @@
...
+ * other - Java LS crashes on WSL Alpine. See [JLS#1612](https://github.com/eclipse/eclipse.jdt.ls/pull/1612).
```
Comment (testforstephen) [inline CHANGELOG.md:7]: Add the link to the original issue https://github.com/redhat-developer/vscode-java/issues/1711

File: CHANGELOG.md (hunk 1 above)
Comment (rgrunber) [reply CHANGELOG.md]: Fixed the typo. Did you want to replace the term "inferSelection" with "infer the selection range" or were you just pointing out specifically where the typo was.

File: CHANGELOG.md (hunk 2 above)
Comment (rgrunber) [reply CHANGELOG.md]: Done.

**Pull request merged**